apps/data-types.md

```diff
@@ -641,6 +641,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | energyLabelsString | string | A list of energy labels applied to this product. Can be one value or a range with two values. |
 | energyLabelsMaxEfficiencyString | string | The highest possible efficiency class of an uploaded energy label. |
 | energyLabelSourceFile | string | A URL with an image or PDF file containing the energy label image supplied by the manufacturer. |
+| productDataSheetSourceFile | string | A URL with an image or PDF file containing the data sheet for the product. |
```

Comment:
@bolsu Since `energyLabelSourceFile` is also listed under ##product and ##productSearch, shouldn't `productDataSheetSourceFile` be mentioned there as well? Or will this possibly be done with another PR?